DOCS-6249 rename (#121)
mulesoft/docs-api-community-manager: modules/ROOT/pages/publish-apis.adoc

== Add APIs from Exchange to Your Community

> . Open the API Community Manager control panel and select *Manage APIs*. From this tab, you can manage the APIs in your community, including their metadata and visibility.

Reviewer: GLOBAL one sentence:
```suggestion
. Open the API Community Manager control panel and select *Manage APIs*. 
From this tab, you can manage the APIs in your community, including their metadata and visibility.
```

== Preview APIs

> . In the API Community Manager control panel, click *Community Builder*. Community Builder displays a preview of

Reviewer: ditto:
```suggestion
. In the API Community Manager control panel, click *Community Builder*. 
Community Builder displays a preview of
```

== Make API Versions Visible

. In the API Community Manager control panel, select *Manage APIs*.
. Select *Edit* next to the API version name.
. Set *Visibility* to *Everyone* if you want it to be publicly available, or *Members only* if you want it to be seen only by logged-in members.
. Select *Apply*. Consumer developers can now see this API version in the API carousel on the community site.

== Minor API Updates

> When an API specification or documentation requires non-breaking corrections, such as changing the text, these changes are given a minor version change, such as `3.0.0` to version `3.0.1`. API Community Manager automatically and immediately imports these updates to all communities in which the API is published, without requiring any action from the API Community Manager administrator.

Reviewer: ```suggestion
When an API specification or documentation requires nonbreaking corrections, such as changing the text, these changes are given a minor version change, such as from `3.0.0` to `3.0.1`. API Community Manager automatically and immediately imports these updates to all communities in which the API is published, without requiring any action from the API Community Manager administrator.
```

. The developer selects *Publish*. The previous API version remains in all communities where it was published
without any changes.

The API Community Manager community administrator must then review and publish the changes:

> . In the API Community Manager control panel, select *Manage APIs*. The previous API version remains unchanged in

Reviewer: one sentence:
```suggestion
. In the API Community Manager control panel, select *Manage APIs*. 
The previous API version remains unchanged in
```
